docs/tutorial/using-native-node-modules.md

```diff
@@ -58,10 +58,14 @@ where to download headers and which version to use:
 
 ```bash
 $ cd /path-to-module/
-$ HOME=~/.electron-gyp node-gyp rebuild --target=0.29.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
+$ HOME=~/.electron-gyp node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
+#On Windows Try this
+cd /path-to-module/
+X:\path-to-module>node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
```

Comment:
I'm good separating the commands for POSIX and Windows cmd, like what we did in Quick Start.